apidef/schema.go

```diff
@@ -62,6 +62,9 @@ const Schema = `{
         },
         "enable_coprocess_auth": {
             "type": "boolean"
+        },
+		"custom_plugin_auth_enabled": {
```

Comment:
Can you file an issue to move the schema into a .json file and an embedfs?

Reply:
yes

Reply:
https://tyktech.atlassian.net/browse/TT-7617